nxdrive/commandline.py

```diff
@@ -739,6 +742,8 @@ def __enter__(self) -> "HealthCheck":
             if self.crash_file.is_file():
                 log.warning("It seems the application crashed at the previous run 😮")
                 State.has_crashed = True
+                with open(str(self.crash_file), "r") as my_file:
```

Comment:
You should leverage the `pathlib` power here:
```python
State.crash_details = self.crash_file.read_text().strip()
```
Also, to be safe, you should surround that new code with a try/except or `contextlib.suppress()` to prevent `OSError`s from starting the app.